gui/slick/interfaces/default/home.mako

```diff
@@ -814,6 +818,15 @@ $(document).ready(function(){
             </script>
         </td>
 
+        <% show_size = 0 %>
+        % try:
+            <% show_size = sickbeard.helpers.get_size(curShow.location) %>
+        % except sickbeard.exceptions.ShowDirNotFoundException:
+            <% show_size = sickbeard.helpers.get_size(curShow._location) %>
+        % endtry
+
```

Comment:
I took the code from [here](https://github.com/SiCKRAGETV/SickRage/blob/master/sickbeard/webserve.py#L1239) for the show location and [here](https://github.com/SiCKRAGETV/SickRage/blob/master/gui/slick/interfaces/default/displayShow.mako#L252) for the display.
So maybe that check needs to be added there too?